.travis.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       node_js: lts/*
       env: semantic-release
       script: npm run semantic-release
-    - node_js: lts/*
+    - node_js: 10
```

Comment:
wait, it looks like coverage is already run only once per build, I don’t think that your change is making any difference 

https://github.com/nock/nock/blob/5c67b46ed2ed74abed05430eb3991332c35dd241/.travis.yml#L36-L40

Are you sure we post coverage multiple times per build/